### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by TaiwoSoftware** [Cyfrin/foundry-full-course-cu#2992] [General]
Hello senior devs my name is Taiwo, I'm a fronten developer 
please i just need an advice i really love frontend development but now that web 3 is everywhere what part of web 3 do you think i should start with

**Comment by EngrPips** [comment]
Hello @TaiwoSoftware, Start with `Blockchain Basics` on Cyfrin Updraft. You at least need to know enough about web3 and smart contracts before you can start building front-end for smart contracts.
